test/600_proxy_docker_py_test.sh

```diff
@@ -5,6 +5,7 @@
 start_suite "Run docker-py test suite against the proxy"
 
 docker_on $HOST1 pull joffrey/docker-py >/dev/null
+docker_on $HOST1 pull busybox >/dev/null
```

Comment:
probably worth adding a comment that this is a workaround for https://github.com/docker/docker-py/issues/745